File: DataStructures/ArraysAndStrings/IsUnique/IsUnique.py


def IsUnique(string):

	for s in string:
		charOcc = 0;


		for t in string:
			if charOcc >= 2:
				break
			if s == t:
				charOcc = charOcc + 1

		if charOcc > 1:
			return False

	return True


if __name__ == '__main__':
	print("Is unique: %s"%IsUnique("asduf"))

File: DataStructures/ArraysAndStrings/IsUnique/IsUnique.js


function isUnique(string){
	for(i=0; i<string.length; i++){
		nextChar = string.charAt(i);
		charOcc = 0;

		for(j=0; j<string.length; j++){
			if(charOcc>=2){
				break;
			}
			if(nextChar==string.charAt(j)){
				charOcc++;
			}
		}
		if(charOcc>1){
			return false;
		}
	}
	return true;
}

console.log("Is Unique: "+isUnique("asdfhjj"));